src/test/librgw_file_aw.cc

```diff
@@ -273,6 +273,11 @@ TEST(LibRGW, DELETE_OBJECT) {
     int ret = rgw_unlink(fs, bucket_fh, object_name.c_str(),
 			 RGW_UNLINK_FLAG_NONE);
     ASSERT_EQ(ret, 0);
+    if (do_create) {
```

Comment:
@mattbenjamin actually, here is alreadly inside an if (do_delete) {} block, this check is to see whether the bucket is created by '--create' option automatically, if not we'd better keep it there.